concourse/scripts/create_concourse_cert.sh

```diff
@@ -19,15 +40,39 @@ if [ "${cert_status}" = "ISSUED" ]; then
   exit 0
 fi
 
-if [ "${created_cert}" = "false" ]; then
-  echo "Resending validation email for ${concourse_fqdn} certificate"
-  aws acm resend-validation-email --certificate-arn "${arn}" --domain "${concourse_fqdn}" --validation-domain "${SYSTEM_DNS_ZONE_NAME}"
+# The validation records are not returned for a couple of seconds from the API after creation
+echo "Getting DNS validation records"
+for _ in $(seq 20); do
+  sleep 3
+
+  cert_info=$(aws acm describe-certificate --certificate-arn "${arn}" --query 'Certificate')
+  dns_validation_record=$(echo "${cert_info}" | jq -r '.DomainValidationOptions[0].ResourceRecord.Name')
```

Comment:
> `jq`

I don't think `jq` is available in the container so not sure this will work.

```
14:12 $ docker run -it governmentpaas/awscli:895cf6752c8ec64af05a3a735186b90acd3db65a jq
Unable to find image 'governmentpaas/awscli:895cf6752c8ec64af05a3a735186b90acd3db65a' locally
895cf6752c8ec64af05a3a735186b90acd3db65a: Pulling from governmentpaas/awscli
957bc456a443: Pull complete
ff683ce0bd24: Pull complete
6ebc75f7cfa5: Pull complete
Digest: sha256:af3e84a701fca875491e064d38860ebadafef1eaee33346a9ec5ae07bd4e8dc6
Status: Downloaded newer image for governmentpaas/awscli:895cf6752c8ec64af05a3a735186b90acd3db65a
docker: Error response from daemon: oci runtime error: container_linux.go:265: starting container process caused "exec: \"jq\": executable file not found in $PATH".
```

Reply:
Ahhh, good catch, I'll add it to the Docker image.

Reply:
@chrisfarms PR created for jq: https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/105, please review and merge first